howto/README.md

```diff
@@ -0,0 +1,49 @@
+# How do I ...
+
```

Comment:
I learned recently that `RST` files get an automatically generated table of contents. It's pretty cool!